restapi_test.go

```diff
@@ -250,6 +251,27 @@ func TestChannelMessageSend2(t *testing.T) {
 	}
 }
 
+// TestChannelMessageSendComplex_files tests sending files via
+// ChannelMessageSendComplex() function. This should not return an error.
+func TestChannelMessageSendComplex_files(t * testing.T) {
+
+	if envChannel == "" {
+		t.Skip("Skipping, DG_CHANNEL not set.")
+	}
+
+	if dg == nil {
+		t.Skip("Skipping, dg not set.")
+	}
+
+	_, err := dg.ChannelMessageSendComplex(envChannel, &MessageSend{
+		Content: "Test sending files with message",
+		File: &File{Name: "test.txt", Reader: strings.NewReader("file content")},
```

Comment:
The should use the `Files` field, not `File`; the latter is simply in the library for backwards compatibility and may be removed in the future.